src/indicators/drought/iri_rainfallforecast.py

```diff
@@ -4,89 +4,117 @@
 from pathlib import Path
 import rasterio
 import xarray as xr
-import rioxarray
+import rioxarray  # noqa: F401
```

Comment:
What happens here if you don't import rioxarray?

Reply:
As discussed, this was an error made up in my head, now removed the unused rioxarray imports